bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/provider/ShellyChannelDefinitions.java

```diff
@@ -344,10 +344,10 @@ public static Map<String, Channel> createRelayChannels(final Thing thing, final
         }
 
         // Shelly 1/1PM and Plus 1/1PM Addon
-        addChannel(thing, add,
```

Comment:
What I am not really clear on here, @jlaur, is which value actually is null and causes the NPE...?
Is it the `relayNum` and thus the `relayNum +1` fails with the NPE?

Reply:
`profile.settings.extSwitch` could be null, and it would only be checked for third parameter, but it was also used in last parameter.

Reply:
Ah, right, thanks. 👍

Reply:
(same for `profile.settings.extSwitch.input0` and `profile.settings.extSwitch.input0.relayNum`)